scripts/integration/docker-compose.axiom.yml

```diff
@@ -0,0 +1,62 @@
+version: "3"
+
+services:
+  postgres:
+    image: postgres:13-alpine
+    environment:
+      POSTGRES_USER: axiom
+      POSTGRES_PASSWORD: axiom
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+  axiom-db:
+    image: axiomhq/axiom-db:latest
+    restart: unless-stopped
+    environment:
+      AXIOM_STORAGE: "file:///data"
+      AXIOM_POSTGRES_URL: "postgres://axiom:axiom@postgres?sslmode=disable&connect_timeout=5"
+    depends_on:
+      - postgres
+  axiom-core:
+    image: axiomhq/axiom-core:latest
+    restart: unless-stopped
+    environment:
+      AXIOM_POSTGRES_URL: "postgres://axiom:axiom@postgres?sslmode=disable&connect_timeout=5"
+      AXIOM_DB_URL: "http://axiom-db:80"
```

Comment:
This should be :8080

Reply:
That certainly looks like it should make sense - testing...

Reply:
Yeah that's definitely it, it shows that it's listening on `8080` here:
```jsonm
{"level":"info","_time":"2022-06-27T16:37:08.778968051Z","caller":"db/serve.go:599","message":"Started listening","service":"db","address":"[::]:8080"}
```

We've changed that recently, that why it started breaking. @spencergilbert I'm pushing a fix